src/support/err.c

```diff
@@ -683,18 +683,41 @@ __wt_ext_strerror(WT_EXTENSION_API *wt_api, WT_SESSION *wt_session, int error)
  *     Progress message.
  */
 int
-__wt_progress(WT_SESSION_IMPL *session, const char *s, uint64_t v)
+__wt_progress(WT_SESSION_IMPL *session, WT_VERBOSE_CATEGORY category, const char *s, uint64_t v)
 {
     WT_DECL_RET;
     WT_EVENT_HANDLER *handler;
     WT_SESSION *wt_session;
+    size_t remain;
+    char operation[1024];
+    const char *msg;
+    bool is_json;
 
     wt_session = (WT_SESSION *)session;
+
     handler = session->event_handler;
+    is_json = FLD_ISSET(S2C(session)->json_output, WT_JSON_OUTPUT_PROGRESS);
+    msg = s == NULL ? session->name : s;
```

Comment:
Nitpick: I'd prefer some brackets here: `msg = (s == NULL) ? session->name : s;`

Reply:
I agree it increases readability. But there are many other places in the code where we do the same kind of assignment with no brackets, I guess I just followed the pattern. If anyone else feels strongly about it, I will add brackets :)